configure.ac

```diff
@@ -169,7 +169,6 @@ AS_IF([test x"$hardening" != x"no"], [
 
   add_hardened_c_flag([-Wformat])
   add_hardened_c_flag([-Wformat-security])
-  add_hardened_c_flag([-Wstack-protector])
```

Comment:
PM

Reply:
I was misunderstanding the OS I was building on, it was exporting `CFLAGS` in the environment and overriding/conflicting with our `EXTRA_CFLAGS`.